command/server/config.go

```diff
@@ -419,6 +422,21 @@ func (c *Config) Merge(c2 *Config) *Config {
 		result.EnableResponseHeaderRaftNodeID = c2.EnableResponseHeaderRaftNodeID
 	}
 
+	result.UnsafeCrossNamespaceIdentity = c.UnsafeCrossNamespaceIdentity
+	if c2.AllowAuditLogPrefixing {
```

Comment:
what does it have with your change tbh?
```suggestion
	if c2.UnsafeCrossNamespaceIdentity {
```

Reply:
Oopsie, fixed. I added `UnsafeCrossNamespaceIdentity` as well because it was missing here.